test/index.ts

```diff
@@ -1,207 +1,26 @@
-import axios from 'axios';
-import { expect } from 'chai';
 import * as dotenv from 'dotenv';
-import { UserInput } from '../src/schema/interfaces';
 import { startServer } from '../src/server';
 import { AppDataSource, clearDB } from '../src/data-source';
-import { User } from '../src/entity/User';
 
-const endpoint = 'http://localhost:3000/';
+export { hashString } from '../src/schema/resolvers';
```

Comment:
Precisa desse export? Quem for usar essa função poderia importar direto de lá, não?